Makefile

```diff
@@ -49,17 +50,3 @@ test-flake8: virtualenv
 .PHONY: test-unit
 test-unit: virtualenv
 	${VIRTUALENV_ROOT}/bin/py.test ${PYTEST_ARGS}
-
-# Build and tag Docker image.
-.PHONY: docker-build
```

Comment:
Let's not lose this tho eh :-)

Reply:
The reason for removing these was because they were incorrectly referring to digital marketplace's Docker Hub account, and we haven't really thought about how we're going to be using Docker for this project anyway. We can always bring back Docker specific commands later once we're at that stage.

Reply:
Ah right I assuemed they were something you worked on the other day. All good anyway, def not for this PR.

Reply:
We can fish it out of the git history if/when we need it later, it's fine